src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -201,8 +201,8 @@ public void AssertCommercialBuild (bool fail = false)
 
 		protected static void AssertDexToolSupported (string dexTool)
 		{
-			if (Builder.UseDotNet && dexTool == "dx") {
-				Assert.Ignore ("dx is not supported in .NET 5+");
+			if (dexTool == "dx") {
+				Assert.Ignore ("dx is not supported");
 			}
 		}
```

Comment:
We could delete this method instead and test parameters for `dx`, but I can do that after this goes in.